docs/troubleshooting/node-syncing.md

```diff
@@ -6,37 +6,39 @@ slug: /troubleshooting/node-syncing
 
-When you port forward you want to allow outside IP addresses to connect to your main node (computer) through port 8444 to the Chia software.
+When you enable port forwarding, you are allowing any system on the Internet to connect to your Chia node through port 8444 to the Chia software.
+
+## Port Forwarding Settings
 
 Most routers will ask you from where you are allowing and to what you are connecting to. You want to set up port forwarding to allow any outside connection to connect to the IP address of your main node on your network through port 8444. Router manufacturers might call the settings different things, but the concept is always the same: Outside computers connecting through port 8444 to your computer.
 
 Here are the settings most routers will ask for:
 
 - Set connection type to _TCP_ or _TCP & UDP_
-- Destination (or Forwarding) IP address - This is your main node (computer) IP address on your internal network; search online on how to do this for your type of computer. If you search for "what is my IP address" it will give you your external IP address, this is not the one you want.
-- Originating (or From) IP address - Set this to all or sometimes just an asterisk may be used `*`
+- Destination (or forwarding) IP address - This is your main node (computer) IP address on your internal network; search online on how to do this for your type of computer. If you search for "what is my IP address" it will give you your external IP address, this is not the one you want.
+- Originating (or from) IP address - Set this to all or sometimes just an asterisk may be used `*`
 
-## Why forward port 8444?
+## Why Forward Port 8444?
 
-All peers (computers) with a closed port 8444 are completely dependent on pc peers with open port 8444. They are the only PCs they can talk to. If you got 1,000 nodes with an open port 8444, but 20,000 nodes with a closed port 8444, trying to sync, it will only just be able to theoretically have enough IP's estimated 3,000 can sync at a time, while the other wait for another open Chia user with open port 8444. Right now (Mid April '20) it seems that number is even much worse. And it causes a scenario where there just isn't enough open port 8444 peers to serve all the closed port 8444 peers. The only way around this is to ensure that you got an open port 8444.
+All newly added nodes are completely dependent on nodes that are allowing port forwarding, because they are the only nodes in which they can sync with. The more nodes there are that don't allow port forwarding can cause a bottleneck to those nodes that do have it enabled.
 
-If you somehow are able to open up your port 8444 you will quickly have peers connecting to you and have a much easier time to get connections established.
+If you enable port forwarding, your node will sync faster, and you will be helping to ensure the stability and overall health of the Chia network.
 
-## Speed up connecting to nodes
+## How to Speed Up Connecting to Nodes
```

Comment:
can we revert this header back to what it was? Changing these will break links that we have in other repos, the capitalization is fine but it needs to be the same wording :+1: